### ISSUE TITLE
update README.md

### ISSUE THREAD

**Issue by schultzm** [davidebolo1993/NanoR#10]
Hi @davidebolo1993 , 

Thank you for writing this software.  I am following through the readme and up to the point of running NanoR, but I noticed one piece of important information missing.  Before the lines on `?NanoPrepareM`, please add a line `library('NanoR')`. 

Thanks,

Mark

**Comment by schultzm** [comment]
Great!  Whilst I've got you on the line, I also needed to do this as biocLite seems to be deprecated:

```
chooseCRANmirror()
# choose your mirror
install.packages("BiocManager")
BiocManager::install(c("rhdf5", "ShortRead"), suppressUpdates=TRUE, suppressAutoUpdate=TRUE)

```

**Comment by davidebolo1993** [comment]
Yes, @schultzm, this is an alternative. For the R version I used to develop NanoR, BiocManager was not available (it requires R >= 3.5 if I remember correctly) but, generally speaking, the way to go is exactly the same:

```R
install.packages('BiocManager')#instead of source("http://bioconductor.org/biocLite.R")
BiocManager::install(c("rhdf5", "ShortRead"), suppressUpdates=TRUE, suppressAutoUpdate=TRUE) # instead of biocLite(c("rhdf5","ShortRead")suppressUpdates=TRUE, suppressAutoUpdate=TRUE)
```
